lib/routes/account.js

```diff
@@ -258,9 +258,26 @@ module.exports = function (
               }).catch(function (err) {
                 log.error({ op: 'Account.verificationReminder.create', err: err })
               })
+
+              if (tokenVerificationId) {
+                // Log server-side metrics for confirming verification rates
+                log.info({
+                  op: 'account.create.confirm.start',
+                  uid: account.uid.toString('hex')
```

Comment:
The `start` below logs the `tokenVerificationId`. Is there a reason to log it for `/account/login` but not `/account/create`?